pootle/checks.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.core import checks
+from django.utils.translation import ugettext as _
+
+
+# Minimum Translate Toolkit version required for Pootle to run.
+TTK_MINIMUM_REQUIRED_VERSION = (1, 11, 0)
+
+# Minimum Django version required for Pootle to run.
+DJANGO_MINIMUM_REQUIRED_VERSION = (1, 7, 6)
+
+# Minimum lxml version required for Pootle to run.
+LXML_MINIMUM_REQUIRED_VERSION = (2, 2, 2, 0)
+
+# Minimum Redis server version required.
+# Initially set to some minimums based on:
+# 1. Ubuntu 12.04LTS's version 2.8.4 (10.04LTS was too old for RQ)
+# 2. RQ requires >= 2.6.0, and
+# 3. Wanting to insist on at least the latest stable that devs are using i.e.
+#    2.8.* versions of Redis
+REDIS_MINIMUM_REQUIRED_VERSION = (2, 8, 4)
+
+
+# XXX List of manage.py commands not to run the rqworker check on.
+# Maybe tagging can improve this?
+RQWORKER_WHITELIST = [
+    "start", "initdb", "revision", "sync_stores", "run_cherrypy",
+    "refresh_stats", "update_stores", "calculate_checks", "retry_failed_jobs"
+]
+
+
+
+@checks.register()
+def test_library_versions(app_configs=None, **kwargs):
```

Comment:
@jleclanche I still think all these functions must be named `check_xxxx`. They are used for checks, not for tests.